configs: defconfig: add CONFIG_ID_EEPROM

RadxaMitchell opened the pull request:
The added CONFIG_ID_EEPROM defined files are:
- radxa-e52c-rk3588s_defconfig
- radxa-e54c-rk3588s_defconfig
- rock-5b-plus-rk3588_defconfig
- rock-5c-rk3588s_defconfig
- rock-5d-rk3588s_defconfig
- rock-5t-rk3588_defconfig

Because it is not feasible to define CONFIG_ID_EEPROM directly in defconfig, need to add a switch defining CONFIG_ID_EEPROM in Kconfig under the evb_RK3588 directory.

RadxaYuntian commented:
5A和5 ITX应该也可以打开，这样他们的镜像在上述系统启动时也可以工作，但在自己上运行时由于硬件检测无法识别，会使用平台默认的dtb。
理论上所有产品都应该可以打开，除了5B会浮动成别的产品的值所以才不能默认都打开。

RadxaYuntian commented:
另外在主线上这个选项是加到[common/Kconfig](https://github.com/u-boot/u-boot/commit/d7d40f614d5e6b6c7369ebd001d3ac191b3e360b#diff-7bd0e5bf81bab2185eb30a4e6fdf7cb46a558fbfab2b07331ecc55fd834b639c)里面的

RadxaMitchell committed:
configs: defconfig: add CONFIG_ID_EEPROM

Because it is not feasible to define CONFIG_ID_EEPROM directly in defconfig,
need to add a switch defining CONFIG_ID_EEPROM in Kconfig under the commonn directory.

The added CONFIG_ID_EEPROM defined files are:
- radxa-e52c-rk3588s_defconfig
- radxa-e54c-rk3588s_defconfig
- rock-5b-plus-rk3588_defconfig
- rock-5c-rk3588s_defconfig
- rock-5d-rk3588s_defconfig
- rock-5t-rk3588_defconfig
- radxa-cm5-io-rk3588s_defconfig
- radxa-cm5-rpi-cm4-io-rk3588s_defconfig
- rock-5-itx-rk3588_defconfig
- rock-5a-rk3588s_defconfig

Signed-off-by: Mitchell Ma <machuang@radxa.com>

RadxaMitchell commented:
5B 和 NX5 的没有添加，原因是 NX5 和 5B 一样 hw_id 的值会浮动
